extension/test/unit/validator-builder.spec.js

```diff
@@ -220,4 +220,21 @@ describe('Validator builder', () => {
       MISSING_REQUIRED_FIELDS_CTP_PROJECT_KEY
     )
   })
+
+  it('on having one random custom field should return error object', () => {
+    const invalidPayment = {
+      custom: {
+        fields: {
+          randomCustomField: 'bar',
```

Comment:
could not get the idea in here, is this possible by API, should be a 400 right ?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/542/commits/0ddd882ef2252fe921d3343025b04995a0944dd1